app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,21 @@
+from .elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(
+            self,
+            nickname: str,
+            musical_instrument: str,
+            bow_level: int
+    ) -> None:
+        super().__init__(nickname, musical_instrument)
+        self.bow_level = bow_level

```

Comment:
Make sure that you store arguments in the protected attributes as mentioned in the checklist